.env.example

```diff
@@ -1,7 +1,10 @@
 SECRET_KEY=insecure-secret-key
 LOGIN_CODE_SECRET=insecure-login-secret-key
 
+# Relative (or absolute if starts with slash) base directory
+REL_BASE_DIR="/app"
 APP_ENV=dev
+APP_DEBUG=True
```

Comment:
Just a check: did you intend to leave debugging on by default?

Reply:
For dev yep.